openmdao/docs/feature_reference/building_blocks/solvers/backtracking/bounds_enforce.rst

```diff
@@ -3,12 +3,73 @@
 Bounds Enforce
 **************
 
-Description of BoundsEnforce.
+The BoundsEnforce linesearch only backtracks until variables that violate their upper and lower bounds.
 
 BoundsEnforce Options
 ----------------------
 
 .. embed-options::
     openmdao.solvers.linesearch.backtracking
     BoundsEnforceLS
-    options
\ No newline at end of file
+    options
+
```

Comment:
For consistency with other docs,  should add the title that Jubs had me institute...

BoundsEnforce Options Examples
----------------------------------

same comment for Armijo-Goldstein

Reply:
what syntax are you using for the underline?

Reply:
same as the header above it, dashes